### DISCUSSION TITLE
Reports > Category -- [reports.category.month] not found

### DISCUSSION THREAD

**Discussion by bartekd123** [firefly-iii/firefly-iii#4547] [General]
I am not sure if its something with just my environment, so wanted to see if others get the same error. Can someone go to reports and run a category report and see if there is an error. For mine, I select all categories, and all accounts, everything default, and I get the following error in the browser:

`Could not render report view: View [reports.category.month] not found.`

I am running the latest version (5.5.1) in Docker. Here are my docker logs:

```
10.150.3.6 - - [28/Mar/2021:14:19:44 -0400] "POST /reports HTTP/1.1" 302 3062 "https://firefly.domain.com/reports" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.1 Safari/605.1.15"
[2021-03-28 18:19:44] local.ERROR: Cannot render reports.category.month: View [reports.category.month] not found.  
10.150.3.6 - - [28/Mar/2021:14:19:44 -0400] "GET /reports/category/3,5,19,83,39,90,74,81,9,2,20,21,37,38,49,51,11,72,54,1,17/37,46,39,6,7,41,12,5,9,2,14,1,40,45,42,32,17,43,34,4,10,48,22,11,3,28,33,19,13/20210301/20210331 HTTP/1.1" 200 2590 "https://firefly.domain.com/reports" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.1 Safari/605.1.15"
```

This report worked in the past, but the last few versions it hasn't for me.

**Comment by kutagh** [comment]
Can report that I encountered the same exact error. I'm running the V2 version of the UI on 5.5.1, how about you?

**Comment by JC5** [comment]
That's a bug, nice find. I'll pick it up for 5.5.2. :+1:

**Reply by bartekd123** [reply]
Nice, thanks!
